### PR TITLE
fixes for bugs noticed during usage

### PR DESCRIPTION
**Pull request by oliverrahner**
Kind of like #1, I am noticing some things during usage of AliceCLI and I'm trying to fix them. E.g. the "Please insert SD-card first" error noted in #1 has been fixed.

I'll add more as I see it.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -50,7 +50,8 @@
 		'pytest~=6.2.5',
 		'coverage~=6.2',
 		'pytest-cov~=3.0.0',
-		'coveralls~=3.3.1'
+		'coveralls~=3.3.1',
+		'tomli=2.0.0'
```
Comment (Psychokiller1888) [inline setup.py:54]: mmh? What requires tomli?

**Comment by Psychokiller1888** [comment]
Hey, thank you so much for this, really neat! Tbh, I did not take the time to study the balena cli and I should have, thank you for that!

File: setup.py (hunk 1 above)
Comment (oliverrahner) [reply setup.py:54]: I am not too versed in Python packaging, but this is the message I get when trying to start `alice` from a fresh venv after doing `pip install --editable .`:

```
(venv) user@box:/usr/src/AliceCLI-orig# alice
Traceback (most recent call last):
  File "/usr/src/AliceCLI-orig/venv/bin/alice", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3252, in <module>
    def _initialize_master_working_set():
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3235, in _call_aside
    f(*args, **kwargs)
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3264, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/src/AliceCLI-orig/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 786, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'tomli; extra == "toml"' distribution was not found and is required by coverage

```

**Comment by oliverrahner** [comment]
The only case I'm using balena for that you didn't use was to locate the SD card device. For that, I only use the same command that you have already been using: `balena util available-drives`. 
I then use a more verbose (and also more stable) output from lsblk to increase the chance to actually find the right device (one with a vfat fs that is actually labeled 'boot') or have the user select one if multiple are matching.

File: setup.py (hunk 1 above)
Comment (oliverrahner) [reply setup.py:54]: Here it is:

```
# pipdeptree -p toml -r
toml==0.10.2
  **- pytest==6.2.5 [requires: toml]**
    - projectalice-cli==1.2.0 [requires: pytest~=6.2.5]
    - pytest-cov==3.0.0 [requires: pytest>=4.6]
      - projectalice-cli==1.2.0 [requires: pytest-cov~=3.0.0]
```

Comment (Psychokiller1888) [reply setup.py:54]: That's.... weird, coverage should install it as it is its own dep. Whatever, not a big deal :-)

**Comment by Psychokiller1888** [comment]
You just let me know when you want it merged? I saw you had it as a draft

**Comment by oliverrahner** [comment]
Yes, that was on purpose. But I think it can be merged now, my own setup is now so far through that I don't need the CLI currently.

**Comment by Psychokiller1888** [comment]
@oliverrahner 

What IDE are you using? There's a lot of warnings and stuff to be corrected

**Comment by oliverrahner** [comment]
Mmh… as these started out as small changes I did the edits directly on a naked Linux machine in nano 😂 but as no linter etc. was in use I’m really wondering where these warnings come from.

I’ll try to correct them in the next days using PyCharm.

**Comment by Psychokiller1888** [comment]
Na, I already pushed the cleanup/fix.Sent from my Galaxy
-------- Original message --------From: Oliver Rahner ***@***.***> Date: 1/14/22  21:02  (GMT+01:00) To: project-alice-assistant/AliceCLI ***@***.***> Cc: Psycho ***@***.***>, State change ***@***.***> Subject: Re: [project-alice-assistant/AliceCLI] fixes for bugs noticed during usage (PR #8) 

Mmh… as these started out as small changes I did the edits directly on a naked Linux machine in nano 😂 but as no linter etc. was in use I’m really wondering where these warnings come from.



I’ll try to correct them in the next days using PyCharm.

**Pull request merged**